conf/config-default.yaml

```diff
@@ -193,6 +193,7 @@ plugins:                          # plugin list
   - proxy-mirror
   - request-id
   - hmac-auth
+  - error-log-logger
```

Comment:
Do not need to be turned on by default.

`# - error-log-logger`, should be fine.